### DISCUSSION TITLE
What is the OpenThread Memory Usage on SAMR21?

### DISCUSSION THREAD

**Discussion by cadamlamb** [openthread/openthread#7594] [Q&A] [answered]
Hi All,

I'm not instantly getting the stack to build on Windows, so instead of pounding my head against the wall for another couple of hours, I thought I would ask what the RAM and Flash memory usage is for this minimal version of the OT stack.

Would that be easy for someone to give me a couple of rough numbers?

Thanks so much everyone!!
Adam

**Comment by jwhui** [comment, marked as answer]
Default build on https://github.com/openthread/ot-samr21/commit/f92d10a2fea0bdefa4a48ab3baf846b9d87f0d26:
```
# arm-none-eabi-size build/bin/ot-cli-mtd 
   text	   data	    bss	    dec	    hex	filename
 125420	      0	  11808	 137228	  2180c	build/bin/ot-cli-mtd
# arm-none-eabi-size build/bin/ot-cli-ftd
   text	   data	    bss	    dec	    hex	filename
 189148	      0	  16064	 205212	  3219c	build/bin/ot-cli-ftd
```

**Reply by cadamlamb** [reply]
That is super helpful; exactly what I was looking for.  Thanks so much!
